examples/torch/maml_ppo_half_cheetah_dir.py

```diff
@@ -48,6 +50,14 @@ def maml_ppo(ctxt, seed, epochs, rollouts_per_task, meta_batch_size):
 
     max_path_length = 100
 
+    tasks = SetTaskSampler(lambda: GarageEnv(
+        normalize(HalfCheetahDirEnv(), expected_action_scale=10.)))
```

Comment:
is `expected_action_scale` necessary `HalfCheetahDirEnv`, or just for `MAML`?

Reply:
I used `expected_action_scale=10.` for both `HalfCheetah` and Metaworld when I did the performance evaluation of MAML.